labs.yml

```
@@ -5,11 +5,14 @@ install:
   min_runtime_version: 13.3
   require_running_cluster: false
   require_databricks_connect: true
+  script: src/databricks/labs/remorph/install.py
+uninstall:
```

can you check the YML formatting, this seems off
```suggestion
    uninstall:
```

---

It's at the same level as `install` and it's picking up correctly.